### PR TITLE
Project cleanup after API/RI split.

### PR DESCRIPTION
**Pull request by Tomas-Kraus**
Still need to update API dependency.
```
Manifest-Version: 1.0
Bundle-Description: Angus Activation Registries Implementation
Bundle-DocURL: https://www.eclipse.org
Bundle-License: http://www.eclipse.org/org/documents/edl-v10.php
Bundle-ManifestVersion: 2
Bundle-Name: Angus Activation Registries
Bundle-SymbolicName: angus-activation
Bundle-Vendor: Eclipse Foundation
Bundle-Version: 1.0.0.SNAPSHOT
Created-By: Apache Maven Bundle Plugin
Export-Package: com.sun.activation.registries;uses:="jakarta.activation,
 jakarta.activation.spi";version="1.0.0"
Extension-Name: org.eclipse.angus
Implementation-Build-Id: master-d911b65
Implementation-Title: Angus Activation Registries
Implementation-Vendor: Eclipse Foundation
Import-Package: jakarta.activation;version="[2.0,3)",jakarta.activation.
 spi;version="[2.0,3)"
Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
Specification-Title: Jakarta Activation Specification
Specification-Vendor: Eclipse Foundation
Specification-Version: 2.1
```

### CODE REVIEW

File: activation-registry/src/main/javadoc/doc-files/speclicense.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<html>
```
Comment (lukasj) [inline activation-registry/src/main/javadoc/doc-files/speclicense.html]: this file does not belong to implementations; remove it

File: pom.xml [hunk 2]
```diff
@@ -18,17 +18,35 @@
...
-    <name>Angus Activation project</name>
+    <name>Angus Activation Project</name>
     <description>${project.name}</description>
     <url>https://github.com/eclipse-ee4j/angus-activation</url>
 
+    <properties>
+        <spec.version>2.1</spec.version>
+        <spec.title>Jakarta Activation Specification</spec.title>
+        <activation.version>2.1.0-RC1</activation.version>
+        <activation.recompile.skip>false</activation.recompile.skip>
```
Comment (lukasj) [inline pom.xml]: this property is no longer needed, remove it as well as its usages

File: pom.xml [hunk 3]
```diff
@@ -18,17 +18,35 @@
...
+        <!-- for the osgiversion-maven-plugin -->
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <javadoc.title>Jakarta Activation API documentation</javadoc.title>
```
Comment (lukasj) [inline pom.xml]: This project is angus activation

File: pom.xml [hunk 4]
```diff
@@ -96,37 +91,116 @@
...
-            deactivating a profile.  This profile purposely has none
-            of the modules I don't want to be deployed.
-        -->
-        <profile>
-            <id>oss-release</id>
-        </profile>
-    </profiles>
+    <dependencyManagement>
+        <dependencies>
+            <!--Our artifacts-->
```
Comment (lukasj) [inline pom.xml]: drop this comment, it is not our artifact

File: pom.xml [hunk 5]
```diff
@@ -382,7 +403,7 @@
                     <header><![CDATA[<br>Jakarta Activation API v${project.version}]]></header>
                     <bottom>
                         <![CDATA[
-Comments to : <a href="mailto:jaf-dev@eclipse.org">jaf-dev@eclipse.org</a>.<br>
+Comments to : <a href="mailto:angus-dev@eclipse.org">angus-dev@eclipse.org</a>.<br>
 Copyright &#169; 2019, ${current.year} Eclipse Foundation. All rights reserved.<br>
 Use is subject to <a href="{@docRoot}/doc-files/speclicense.html" target="_top">license terms</a>.]]>
```
Comment (lukasj) [inline pom.xml]: drop this "use" sentence, this is not spec project and does not contain "jakarta" packages

File: activation-registry/src/main/javadoc/doc-files/speclicense.html (hunk 1 above)
Comment (Tomas-Kraus) [reply activation-registry/src/main/javadoc/doc-files/speclicense.html]: Removed

File: pom.xml (hunk 2 above)
Comment (Tomas-Kraus) [reply pom.xml]: Removed here and in maven-compiler-plugin config.

File: pom.xml (hunk 3 above)
Comment (Tomas-Kraus) [reply pom.xml]: Fixed.

File: pom.xml (hunk 4 above)
Comment (Tomas-Kraus) [reply pom.xml]: Removed.

File: pom.xml (hunk 5 above)
Comment (Tomas-Kraus) [reply pom.xml]: Fixed. speclicense.html file was removed from the project.

**Review by lukasj — APPROVED** [review]
LGTM

**Pull request merged**